cilium/cmd/cleanup.go

```diff
@@ -40,6 +40,9 @@ working condition after uninstalling the Cilium agent.`,
 }
 
 var (
+	vp      *viper.Viper              = viper.New()
+	regOpts *option.RegisteredOptions = option.NewRegisteredOptions(vp)
+
```

Comment:
Nit: these don't seem to be specific for the `cleanup` command (i.e. the `cleanup.go`) but used across several commands in the package. Maybe move them to `root.go`?

Reply:
That´s right, I moved it.